tool-labs/accounteligibility/events.php

```diff
@@ -22,6 +22,46 @@ public function getDefaultEventID()
      */
     public function getEvents()
     {
+
+        ##########
+        ## 2017: Wikimedia Foundation elections
+        ##########
+        yield (new Event(44, 2017, 'Wikimedia Foundation elections', '//meta.wikimedia.org/wiki/Wikimedia Foundation elections/2017'))
+            ->addRule(new NotBlockedRule(1), Workflow::HARD_FAIL)// not blocked on more than one wiki
+            ->addRule(new NotBotRule(), Workflow::HARD_FAIL)
+            ->addRule(new EditCountRule(300, null, '<20170401', EditCountRule::ACCUMULATE))// 300 edits before 1 April 2017
+            ->addRule(new EditCountRule(20, '20161001', '<20170401', EditCountRule::ACCUMULATE))// 20 edits between 1 October 2016 and 1 April 2017
+            ->withExtraRequirements(['Your account must not be used by a bot.'])
+            ->withExceptions([
+                'You are a Wikimedia server administrator with shell access.',
+                'You have commit access and have made at least one merged commit in git to Wikimedia Foundation utilized repos between 1 October 2016 and 1 April 2017.',
+                'You are a current Wikimedia Foundation staff member or contractor employed by the Foundation as of 1 April 2015.',
+                'You are a current staff member or contractor employed by an approved Wikimedia Chapter, Thematic Organization or User Group as of 1 April 2017.',
```

Comment:
(or more precisely will be added very shortly as it's getting put on to a couple different templates)